server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -740,6 +749,20 @@ public boolean checkPermission(@NotNull Collection<Long> connectionIds, @NotNull
         return true;
     }
 
+    @SkipAuthorize
+    public BinaryDataResult getBatchImportTemplateFile() throws IOException {
+        String locale = LocaleContextHolder.getLocale().toLanguageTag().toLowerCase();
+        try (InputStream input = MySQLObjectNameUsingReservedWords.class.getClassLoader()
```

Comment:
what is the relationship with MySQLObjectNameUsingReservedWords?

Reply:
totally not, my fault, fixed~